tests/tests_lite/conftest.py

```diff
@@ -0,0 +1,52 @@
+import os
+from typing import List
+
+import pytest
```

Comment:
@awaelchli just as a heads up, you will probably need some of the other items we have in `conftest` for tests to work properly.

Reply:
Yes, I included these already in #14492